PWGCF/EbyEFluctuations/Tasks/CMakeLists.txt

```diff
@@ -43,3 +43,8 @@ o2physics_add_dpl_workflow(factorial-moments
                     SOURCES FactorialMomentsTask.cxx
                     PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore O2Physics::PWGCFCore
                     COMPONENT_NAME Analysis)
+
+o2physics_add_dpl_workflow(netchargefluct
```

Comment:
You will need to use `netcharge-fluctuations` according to the name of your source file and you will need to change the name of this one to `netchargeFluctuations.cxx`